app/views/comment_mailer/comment_notification.text.erb

```diff
@@ -3,6 +3,6 @@
 <% if @comment.ultimate_parent.is_a?(Tag) then %>the tag "<%= @comment.ultimate_parent.commentable_name %>" (<%= url_for(:controller => :tags, :action => :show, :id => @comment.ultimate_parent, :only_path => false) %>) <% else %>"<%= @comment.ultimate_parent.commentable_name.html_safe %>" (<%= polymorphic_url(@comment.ultimate_parent, :only_path => false) %>)<% end %>:
 <%= text_divider %>
 
-<%= to_plain_text(raw @comment.sanitized_content) %>
+<%= to_plain_text(raw(@comment.sanitized_mailer_content))%>
 
 <%= render 'comment_notification_footer', formats: [:text] %><% end %>
```

Comment:
While you're here, would you mind adding the missing final newline to this file?